ddtrace/llmobs/_integrations/openai.py

```diff
@@ -60,7 +60,7 @@ def user_api_key(self, value: str) -> None:
     def trace(self, pin: Pin, operation_id: str, submit_to_llmobs: bool = False, **kwargs: Dict[str, Any]) -> Span:
         base_url = kwargs.get("base_url", None)
         submit_to_llmobs = self.is_default_base_url(str(base_url) if base_url else None) and (
-            operation_id.endswith("Completion") or operation_id == "createEmbedding"
+            operation_id in ("createCompletion", "createChatCompletion", "createEmbedding", "createResponse")
```

Comment:
BTW: this PR only focuses on APM spans, so this change here is unnecessary. Let's move this to the next PR